### PR TITLE
Lenovo license update

### PR DESCRIPTION
**Pull request by amuraleedhar**
SUMMARY

Updating License after review from Lenovo legal team.

ISSUE TYPE

- Documentation Pull Request

COMPONENT NAME

lib/ansible/modules/network/enos/enos_command.py
lib/ansible/modules/network/enos/enos_config.py
lib/ansible/modules/network/enos/enos_facts.py
lib/ansible/plugins/action/enos.py
lib/ansible/plugins/action/enos_config.py
lib/ansible/plugins/cliconf/enos.py
lib/ansible/plugins/terminal/enos.py
lib/ansible/utils/module_docs_fragments/enos.py

ANSIBLE VERSION

ansible 2.5.0
config file = /etc/ansible/ansible.cfg
configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
ansible python module location = /usr/local/lib/python2.7/dist-packages/ansible-2.5.0-py2.7.egg/ansible
executable location = /usr/local/bin/ansible
python version = 2.7.6 (default, Oct 26 2016, 20:30:19) [GCC 4.8.4]

ADDITIONAL INFORMATION

This is an effort to create config modules for enos based switches from Lenovo.

### CODE REVIEW

**Comment by gundalow** [comment]
It's the copyright character, please use `(c)` instead

**Review by gundalow — APPROVED** [review]

**Comment by abadger** [comment]
The two things I see are: (1) Inclusion of "All rights reserved".  This is obsolete verbiage that is just confusing in open source software (as we are licensing the work to give away specific rights).  It became 100% obsolete in 2000: https://en.wikipedia.org/wiki/All_rights_reserved

(2) Removing the copyright notice for Red Hat.  I'm always very careful when I remove copyright notices from a file.  We should check that those files have never had a commit from a Red Hat/Ansible employee before doing that.

File: lib/ansible/utils/module_docs_fragments/enos.py [hunk 1]
```diff
@@ -1,19 +1,14 @@
-# Copyright (C) 2017 Lenovo, Inc.
 #
-# This file is part of Ansible
+# Copyright © 2017 Lenovo. All rights reserved.
```
Comment (gundalow) [inline lib/ansible/utils/module_docs_fragments/enos.py]: `(C)`

**Review by gundalow — CHANGES_REQUESTED** [review]

File: lib/ansible/utils/module_docs_fragments/enos.py (hunk 1 above)
Comment (amuraleedhar) [reply lib/ansible/utils/module_docs_fragments/enos.py]: Done

**Review by gundalow — APPROVED** [review]

**Comment by abadger** [comment]
Looks good.  +1 to merge.

**Comment by gundalow** [comment]
Thanks,
Merged

**Pull request merged**